src/components/Header/SideNav/index.js

```diff
@@ -16,6 +16,7 @@ const SideNavPanel = styled.nav`
   left: 0;
   top: 0;
   z-index: ${props => props.theme.zIndexes.header};
+  overflow-y: scroll;
```

Comment:
Maybe `overflow-y: auto` might be better? 

https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-y says "overflow-y:scroll — always adds a scrollbar" and "overflow-y:auto — on most browser, equivalent to scroll"

Reply:
- [ ] Done 👍 Still works.